### ISSUE TITLE
Zoom event?

### ISSUE THREAD

**Issue by BEagle1984** [gluonhq/maps#87]
I'd like to add a slider over the map to control the zoom level (for improved accessibility etc.). Unfortunately, the map doesn't seem to expose the observable `prefZoom` property nor allow a callback, so it cannot be bi-directionally bound.

Am I right or have I overlooked it?

Is there a reason for this not to be possible? Or would you accept a PR adding this possibility? (In case, what would you prefer? A callback or proxying the `prefZoom` property through the `MapView`?)

**Comment by ctipper** [comment]
I proposed two small methods last year which provide the functions required to embed mapView. [MapView does not update prefCenter and prefZoom #69](https://github.com/gluonhq/maps/issues/69) an observable is obviously the correct way but the developers haven't really been paying attention to this project
